src/GitHub.VisualStudio/AssemblyResolverPackage.cs

```diff
@@ -35,7 +35,6 @@ public class AssemblyResolverPackage : Package
 
         public AssemblyResolverPackage()
         {
-            Debug.WriteLine("GitHub assembly resolver is now active.");
             extensionDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
             AppDomain.CurrentDomain.AssemblyResolve += LoadAssemblyFromExtensionDir;
```

Comment:
The two `ProvideAutoLoad` lines are commented out so this package should never get loaded.

I've only left it here in case we're using an "Experimental" version on top of an old "AllUsers" version. Otherwise the "AllUsers" package might be visible behind the now missing "Experimental" package.

It would make sense to empty out this package completely and only leave an empty stub (that would generally not get loaded). If this unusual situation does happen it can be really confusing with assemblies from different versions loaded.

P.S. I'm testing inline comments on the `Debug` line. ;-)